.taskcluster.yml

```diff
@@ -43,6 +43,7 @@ tasks:
                 git -c advice.detachedHead=false checkout '${head_rev}'
 
                 go install github.com/taskcluster/taskcluster/v51/tools/jsonschema2go/jsonschema2go@latest
+                go install github.com/taskcluster/taskcluster/v51/workers/generic-worker/gw-codegen@latest
```

Comment:
you can delete line 45 here, line 46 can replace it

Reply:
Still need `jsonschema2go` for the go generate of the `generated_types_test.go` file :)

Reply:
my mistake!